source/libs/cache.ts

```diff
@@ -53,6 +59,17 @@ if (location.pathname === '/_generated_background_page.html') {
 
 		const {code, key, value, expiration} = request;
 		if (code === 'get-cache') {
+			if (storage) {
```

Comment:
No fallbacks, it should just use the storage.

Reply:
Purging? Heres what I was thinking:
On every get and/or set operation the purge function is called and does the following:

 - Gets all the data in storage currently
 - Filters out items not created the cache (probably by prefixing the keys)
 - Checks each item and see when it expires, if it has remove it.

We could also use the `Storage​Area​.get​Bytes​InUse` API to retrieve the size of the cache and remove items that haven't been used recently using the `lastUse` key.

Reply:
I don't think we should depend on the available space. When it's expired it should just go regardless.

If we check the entire cache on every get/set that will make each operation heavy. I'd suggest:

- when getting an item, ensure it's not expired. If it is, delete it and return undefined.
- automatically run the purge with a setInterval of 24h